runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePanel.java

```diff
@@ -250,6 +250,18 @@ public void mouseClicked(MouseEvent e)
 						selectedEndPoint = endpoint;
 						updateButtons();
 					}
+
+					@Override
+					public void mouseEntered(MouseEvent e)
+					{
+						panel.setBackground(ColorScheme.DARKER_GRAY_COLOR.brighter().brighter());
```

Comment:
Same thing as before, check if it's possible to use MEDIUM_GRAY_COLOR, it might too bright, but try it.